### DISCUSSION TITLE
Laravel doctrine command line `php artisan  doctrine:generate:entities` has error

### DISCUSSION THREAD

**Discussion by vanchao0519** [doctrine/doctrine2#10350] [Support Questions] [answered]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | yes/no
| Version     | 2.14.x

#### Summary

The problem details show below:
![doctrine-bug-screenshot-01](https://github.com/vanchao0519/laravel-doctrine-bug-report-001/blob/main/assets/images/doctrine-bug-screenshot-01.png
)

#### How to reproduce

```php
php artisan  doctrine:generate:entities ./   --generate-annotations --generate-methods

Generating getter and setter for default entity manager...

TypeError

str_repeat() expects parameter 2 to be int, string given

  at vendor/doctrine/orm/lib/Doctrine/ORM/Tools/EntityGenerator.php:486
    482▕      * @return void
    483▕      */
    484▕     public function setNumSpaces($numSpaces)
    485▕     {
  ➜ 486▕         $this->spaces    = str_repeat(' ',  $numSpaces);
    487▕         $this->numSpaces = $numSpaces;
    488▕     }
    489▕
    490▕     /**

      +15 vendor frames
  16  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

```

**Comment by derrabus** [comment, marked as answer]
If you want to file this issue as a bug report, please provide detailed steps to reproduce your problem using ORM APIs only. Moving to support fort now.

**Comment by vanchao0519** [comment]
Hi @derrabus ! I had edit this post to provide detailed steps to reproduce my problem using ORM APIs only. 😊

**Reply by derrabus** [reply]
`artisian` is not an ORM API.

**Reply by vanchao0519** [reply]
Sorry @derrabus  ! I will fixed this bug to laravel-doctrine project. Please help me to close this discussion
